package.json

```diff
@@ -52,12 +54,11 @@
     "prettier": "^2.1.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "sinon": "^9.2.0",
     "ts-jest": "^26.4.1",
     "typescript": "^4.0.3",
     "webpack": "^4.44.1",
     "webpack-cli": "^4.1.0",
-    "winston": "^3.3.3"
-  },
-  "devDependencies": {}
+    "winston": "^3.3.3",
+    "supertest": "^5.0.0"
```

Comment:
**Comment**: We should be leaving developer dependencies empty, as communicated before.

Reply:
Strange, in the actual JSON package file, devdependencies is empty (not listed). 

Link to JSON file for this branch: https://github.com/martinsung00/thoridal/blob/THOR-002-complete-backend/package.json